basedtyping/__init__.py

```diff
@@ -58,15 +51,13 @@ def __getitem__(self, item):
     Function = Callable
 
 # Unlike the generics in other modules, these are meant to be imported to save you from the boilerplate
-
 T = TypeVar("T")
+out_T = TypeVar("out_T", covariant=True)
+in_T = TypeVar("in_T", contravariant=True)
 T_co = TypeVar("T_co", covariant=True)
 T_contra = TypeVar("T_contra", contravariant=True)
```

Comment:
Why duplicate these? If we want to rename them we should add a note saying the old ones are deprecated

Reply:
yeah, ig